lib/routes/sms.js

```diff
@@ -50,7 +50,7 @@ module.exports = (log, db, config, customs, sms) => {
         const templateName = TEMPLATE_NAMES.get(request.payload.messageId)
         const acceptLanguage = request.app.acceptLanguage
 
-        let phoneNumberUtil, parsedPhoneNumber
+        let phoneNumberUtil, parsedPhoneNumber, region
```

Comment:
No biggie but is there a reason I don't see for moving `region` up to this level? It doesn't look like it's used?

Reply:
Ahha! You are right. I used a different formatter (FormatAsYouType or something like that) to begin with, and that required `region`. Will revert that change.